data/brands/amenity/bank.json

```diff
@@ -14930,6 +14930,28 @@
         "name:en": "Kagoshima Bank",
         "name:ja": "鹿児島銀行"
       }
+    },
+    {
+      "displayName": "АИК банка",
+      "locationSet": {"include": ["rs"]},
+      "matchNames": [
+        "AIK bank",
+        "AIK banka",
```

Comment:
Thank you, but for pure English character, you don't need to add uppercase and lowercase twice in matchname, because when run auto-formatter program they will be transformed to lowercase all.

So I think "AIK BANK" and "AIK BANKA" is not needed. but don't worry and don't need your action, this will be fix automatically.

See https://github.com/osmlab/name-suggestion-index/wiki/Contributing#matchnamesmatchtags-optional